src/lib/navigation.ts

```diff
@@ -161,12 +154,12 @@ export const navigation: NavigationType = [
       //   href: '/test-suites/test-flake-detection',
       //   icon: 'flake',
       // },
-     
     ],
   },
   {
     title: 'Test Runners',
     href: '/test-runners',
+    omitFromDocumentTitle: true,
```

Comment:
This is kind of opinionated but I don't think "Test runners" adds any value to have in the document title. This gives us a way to opt out individual sections.